src/clr-icons/all.umd.ts

```diff
@@ -4,9 +4,12 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import { ApiModule } from './api';
 import { ClrAllSet } from './shapes/all';
+import { ApiModule } from './api';
 
 /* tslint:disable:variable-name */
-const ClarityIcons = ApiModule.instance;
-ClarityIcons.init(ClrAllSet);
+ApiModule.instance.init();
+
+if (window && window.hasOwnProperty('ClarityIcons')) {
+  window.ClarityIcons.add(ClrAllSet);
```

Comment:
I'm not sure I understand why you still need the `init()`. Can't you just do:
``
`typescript
ApiModule.instance.add(ClrAllSet);
```